kmir/src/tests/integration/data/prove-rs/show/local-raw-fail.expected

```diff
@@ -4,11 +4,34 @@
 │   function: main
 │   span: 51
 │
-│  (19 steps)
-└─ 3 (stuck, leaf)
-    rvalueAddressOf ( mutabilityMut , place ( ... local: local ( 3 ) , projection: p
-    function: main
-    span: 50
+│  (93 steps)
+├─ 3
+│   thunk ( #expect ( thunk ( #compute ( binOpEq , thunk ( #compute ( binOpBitAnd ,
+│   function: main
+┃
+┃ (1 step)
+┣━━┓
+┃  │
+┃  ├─ 4
+┃  │   AssertError ( assertMessageMisalignedPointerDereference ( ... required: operandC
+┃  │   function: main
+┃  │
+┃  │  (1 step)
+┃  └─ 6 (stuck, leaf)
+┃      #ProgramError ( AssertError ( assertMessageMisalignedPointerDereference ( ... re
+┃      function: main
+┃
+┗━━┓
+   │
+   ├─ 5
+   │   #execBlockIdx ( basicBlockIdx ( 1 ) ) ~> .K
+   │   function: main
+   │
+   │  (9 steps)
+   └─ 7 (stuck, leaf)
+       #projectedUpdate ( toLocal ( 2 ) , thunk ( rvalueAddressOf ( mutabilityMut , pla
+       function: main
+       span: 53
```

Comment:
These are examples of where we see `thunk(#expect(thunk(...), ...))` causing us to branch into "maybe it could be an assertion error, and maybe not?!?" This allows us to get further into execution (to the assertion branch), without implementing all the dependencies/operators inside.